### PR TITLE
[indexer] Fixed FeaturesLoaderGuard.

### PR DESCRIPTION
**Pull request by ygorshenin**

### CODE REVIEW

File: search/ranker.cpp [hunk 1]
```diff
@@ -252,7 +256,8 @@ class PreResult2Maker
     string name;
     string country;
 
-    LoadFeature(res1.GetId(), ft, center, name, country);
+    if (LoadFeature(res1.GetId(), ft, center, name, country))
```
Comment (mpimenov) [inline search/ranker.cpp]: !

Comment (ygorshenin) [inline search/ranker.cpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Comment by ygorshenin** [comment]
DUDE JTTP

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**